towerlib/towerlib.py

```diff
@@ -2018,3 +2031,176 @@ def get_schedule_by_name(self, name):
 
         """
         return next(self.schedules.filter({'name__iexact': name}), None)
+
+    def update_all_organization_projects(self, organization_name):
+        """Update all the projects in ansible tower for a given organization.
+
+        Args:
+            organization_name: The name of the organization.
+
+        """
+        organization = self.get_organization_by_name(organization_name)
+        if not organization:
+            raise InvalidOrganization(organization_name)
+        outputs = []
+        for project in organization.projects:
+            output = project.update()
+            if not output:
+                self._logger.error(f'{project.name} failed to update')
+            outputs.append(output)
+        return all(outputs)
+
+    def update_project_by_id(self, project_id):
+        """Update the ansible tower project with given project id.
+
+        Args:
+            project_id: The id of the project, which is to be updated.
+
+        Returns:
+            list: List of response of api request as json on success, None otherwise.
+
+        """
+        project = self.get_project_by_id(project_id)
```

Comment:
This needs to be safe, we need to handle the case were the project_id provided is not valid and not results in an actual project.